combat-trainer.lic

```diff
@@ -3889,7 +3889,7 @@ class GameState
 end
 
 before_dying do
-  DRCA.release_cyclics
+  DRCA.release_cyclics unless get_settings.do_not_release_combat_cyclic
```

Comment:
I'm not a fan of using `get_settings` in `before_dying`, especially for `combat-trainer` which people often override with sub-YAMLs. Could we initialize a global during setup?